### DISCUSSION TITLE
Lombok warnings with Java 17

### DISCUSSION THREAD

**Discussion by matassavickisvertex** [spotbugs/spotbugs#1817] [Q&A]
Hello. Our team is using SpotBug, Lombok and Java 11, we used lombok.extern.findbugs.addSuppressFBWarnings = true to ignore Lombok error. But recently we migrated to Java 17 and we started to get SpotBug warnings that is related to Lombok annotations. We still kept the same addSuppressFBWarnings = true flag. Does anyone encounter this issue?

**Comment by ThrawnCA** [comment]
What is the warning?

**Comment by hazendaz** [comment]
closing as no follow up and given saturation of java 17 and lombok at this point its likely no longer an issue.  If it is, please open a ticket rather than discussion.
